tests/fake/test_fake_set_maintenance.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+
+import pytest
+
+from pubtools.pulplib import (
+    FakeController,
+    Repository,
+    FileRepository,
+    MaintenanceReport,
+    InvalidDataException,
+)
+
+OWNER = "%s@%s" % (os.environ.get("USER"), os.environ.get("HOSTNAME"))
```

Comment:
I think you should be overriding these during the test, rather than taking them from the environment (where they might be missing altogether).

You can use monkeypatch.setenv for setting an environment variable for the duration of a test.  It would however require you to adjust the code slightly to read the environment variables more than once (not only at compile time).